plugins/node/instrumentation-fs/package.json

```diff
@@ -42,14 +42,13 @@
     "@opentelemetry/api": "^1.0.0"
   },
   "devDependencies": {
-    "@opentelemetry/api": "1.0.4",
+    "@opentelemetry/api": "^1.0.0",
     "@opentelemetry/context-async-hooks": "1.2.0",
     "@opentelemetry/resources": "1.2.0",
     "@opentelemetry/sdk-trace-base": "1.2.0",
     "@opentelemetry/sdk-trace-node": "1.2.0",
     "@types/mocha": "7.0.2",
     "@types/node": "14.17.9",
-    "@types/shimmer": "1.0.2",
```

Comment:
seem unused

Reply:
shimmer shouldn't be used in any instrumentation directly

Reply:
I know, but it turned out that dns uses types only in tests and `@opentelemetry/plugin-react-load` really uses shimmer.

As far as I remember `@opentelemetry/plugin-react-load` is somewhat different as it doesn't depend on `@opentelemetry/instrumentation`. I think it's what the name tells - a plugin for react.